systemtest/src/main/java/io/strimzi/systemtest/kafkaclients/KafkaClientOperations.java

```diff
@@ -46,4 +48,12 @@
      * @throws Exception exception
      */
     int receiveMessagesTls(long timeoutMs) throws Exception;
+
+    /**
+     * Getting consumer group current offsets
+     * @param timeoutMs timeout in millisecondsß
+     * @return command output
+     * @throws Exception exception
+     */
+    Map<String, String> getCurrentOffsets(long timeoutMs) throws Exception;
```

Comment:
Do you really need it here? In other classes where is the `KafkaClientOperations` class extended you are throwing the `throw new UnsupportedOperationException();` -> wouldn't it be easier to add it just to `InternalKafkaClient`?

Reply:
Agree.